src/dfmt/formatter.d

```diff
@@ -448,7 +448,7 @@ private:
                 write(" ");
             else if (prevTokenEndLine == currTokenLine || (t == tok!")" && peekIs(tok!"{")))
                 write(" ");
-            else if (t == tok!"else")
+            else if (t == tok!"else" || t == tok!"identifier")
```

Comment:
can we change this to
```d
else if (t.among!(tok!"else", tok!"identifier"))
```
with
```d
import std.algorithm : among;
```
so future additions to this will be easier to write?

Reply:
does `peekBackIsOne` the same? or do you prefer `among`?

Reply:
`peekBackIsOneOf` will work too, won't require an extra import then